template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -129,6 +129,13 @@ def get_args():
             "help": (
                 "how many proof jobs marked 'EXPENSIVE' to run in parallel. "
                 "Default: %(default)s"),
+    }, {
+            "flags": ["--target"],
+            "metavar": "T",
+            "default": "_report",
+            "help":
+                "which target to run (see 'Phony Rules' in Makefile.common). "
+                "default: %(default)s ",
```

Comment:
Adding `choices` here could be helpful. This [PR](https://github.com/karkhaz/aws-templates-for-cbmc-proofs/pull/2) would allow you to run `make -f Makefile.common list` and get a sorted list of targets.